Source/Bricks/Generic/GenericBrick.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  GenericBrick.swift
+//  BrickKit
+//
+//  Created by Ruben Cagnie on 1/23/17.
+//  Copyright © 2017 Wayfair. All rights reserved.
+//
+
+import UIKit
+
+protocol ViewGenerator {
+    func generateView(frame: CGRect, in cell: GenericBrickCell) -> UIView
+}
+
+public class GenericBrick<T: UIView>: Brick, ViewGenerator {
+    public typealias ConfigureView = (view: T, cell: GenericBrickCell) -> Void
+
+    public var configureView: ConfigureView
+
+    public init(_ identifier: String = "", size: BrickSize, backgroundColor: UIColor = .clearColor(), backgroundView: UIView? = nil, configureView: ConfigureView) {
+        self.configureView = configureView
+        super.init(identifier, size: size, backgroundColor: backgroundColor, backgroundView: backgroundView)
+    }
+
+    /// Class variable: Default nib name to be used for this brick's cell
+    // If not overriden, it uses the same as the Brick class
+    public override class var nibName: String {
+        let generic = NSStringFromClass(T.self).componentsSeparatedByString(".").last!
+        return "Generic[\(generic)]"
```

Comment:
The nibName is used to identify the brick when registering. So in this case, there won't be a nib loaded, it will use the cellClass (GenericBrickCell)